internal/config/config.go

```diff
@@ -195,6 +198,14 @@ func GetConfig() *Config {
 		GatewayClusterServiceName:                    options.GetString(GATEWAY_CLUSTER_SERVICE_NAME),
 		PrometheusPushGateway:                        options.GetString(PROMETHEUS_PUSH_GATEWAY),
 	}
+
+	if os.Getenv("CLOWDER_ENABLED") == "true" {
+		cfg := clowder.LoadedConfig
+		config.RedisHost = cfg.InMemoryDb.Hostname
+		config.RedisPort = strconv.Itoa(cfg.InMemoryDb.Port)
+		config.KafkaBrokers = []string{fmt.Sprintf("%s:%v", cfg.Kafka.Brokers[0].Hostname, *cfg.Kafka.Brokers[0].Port)}
```

Comment:
Please consider hiding this logic behind a method:
https://github.com/RedHatInsights/app-common-go/issues/11

Reply:
added the KafkaServers method